core/src/main/java/org/elasticsearch/cluster/block/ClusterBlocks.java

```diff
@@ -203,6 +203,29 @@ public ClusterBlockException indicesBlockedException(ClusterBlockLevel level, St
         return new ClusterBlockException(unmodifiableSet(blocks.collect(toSet())));
     }
 
+    public ClusterBlockException indicesAllowReleaseResources(String[] indices) {
```

Comment:
can I ask for some java docs? I get the name `indicesAllowReleaseResources` because I know the context of this change, but I think it's good to give some context as to what it checks.